src/main/resources/org/craftercms/studio/api/v2/dal/ItemDAO.xml

```diff
@@ -602,4 +602,24 @@
             #{path}
         </foreach>
     </update>
+
+    <select id="getContentTypeUsages" resultType="org.craftercms.studio.api.v2.dal.Item">
+        SELECT
+          I.path, I.system_type AS systemType
+        FROM
+          item I
+        INNER JOIN
+          site S
+        ON
+          S.site_id = #{siteId}
+        WHERE
+          I.site_id = S.id
+        AND
+          (
+            (I.system_type IN ('page', 'component') AND I.content_type_id = #{contentType})
```

Comment:
if #{contentType} is a regex, you should switch it to use SQL patterns and use `like` instead of `=`

Same for the I.path below.

Reply:
those are simple strings, not regex